code/modules/surgery/organs/organ_internal.dm

```diff
@@ -129,17 +120,12 @@
 		STOP_PROCESSING(SSobj, src)
 	return ..()
 
-/obj/item/organ/attack(mob/living/carbon/M, mob/user)
-	if(M == user && ishuman(user))
-		var/mob/living/carbon/human/H = user
-		if(status == ORGAN_ORGANIC)
-			var/obj/item/reagent_containers/food/snacks/S = prepare_eat(H)
-			if(S)
-				qdel(src)
-				if(H.put_in_active_hand(S))
-					S.attack(H, H)
-	else
-		..()
+/obj/item/organ/proc/CanEat()
+	if(status == ORGAN_ORGANIC)
```

Comment:
Maybe a new flag to handle this.
The condition then would something like this:
```
if(status == ORGAN_ORGANIC && !(organ_flags & ORGAN_INEDIBLE))
```
Then add the flag to brains and stuff. Using a proc for this seems weird.

Reply:
but there's conditions where you can eat brains